README.md

```diff
@@ -1,55 +1,56 @@
-
-> **WARNING**: This source is solely for demonstrating an architecture proposal. It should not be used to drive cars. 
+A meta-repository for the Autoware architecture proposal feasibility study created by Tier IV. For more details about the architecture itself, please read this [overview](/design/Overview.md).
 
-> **NOTE**: The features in [AutowareArchitectureProposal.iv](https://github.com/tier4/AutowareArchitectureProposal.iv) will be merged into Autoware.Auto.
+> **WARNING**: All source code relating to this meta-repository is solely intended to demonstrate a potential new architecture for Autoware, and should not be used to autonomously drive a real car!
+> 
+> **NOTE**: The features within the [AutowareArchitectureProposal.iv repository](https://github.com/tier4/AutowareArchitectureProposal.iv) are planned to be merged into [Autoware.Auto](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto).
 
-Architecture overview is [here](/design/Overview.md).
-
-# How to setup
+# Installation Guide
 
 ## Requirements
 
 ### Hardware
 
-- x86 CPU (8 or more cores)
-- 16 GB or more of memory
-- Nvidia GPU (4GB or more of memory) \*optional
-  - When you use following packages, GPU is mandatory:
-    - lidar_apollo_instance_segmentation
-    - traffic_light_ssd_fine_detector
-    - cnn_classifier
+- x86 CPU (8 cores)
+- 16GB RAM
+- Nvidia GPU (4GB RAM) \*optional
+  - Although not required to run basic functionality, a GPU is mandatory in order to run the following components:
+    - lidar_apollo_instance_segmentation
+    - traffic_light_ssd_fine_detector
+    - cnn_classifier
 
-If cuda or tensorRT is already installed, it is recommended to remove it.
+> Note that the specifications above are the minimum requirements. Performance will be improved with more cores, RAM and a higher-spec graphics card.
 
 ### Software
 
  - Ubuntu 18.04
  - Nvidia driver
  
-If cuda or tensorRT is already installed, it is recommended to remove it.
+## Review licenses 
+The following software will be installed during the installation process, so please confirm their licenses first before proceeding.
 
+- [CUDA 10.2](https://docs.nvidia.com/cuda/eula/index.html)
+- [cuDNN 7](https://docs.nvidia.com/deeplearning/sdk/cudnn-sla/index.html)
+- [osqp](https://github.com/oxfordcontrol/osqp/blob/master/LICENSE)
+- [ROS Melodic](https://github.com/ros/ros/blob/noetic-devel/LICENSE)
+- [TensorRT 7](https://docs.nvidia.com/deeplearning/sdk/tensorrt-sla/index.html)
+ 
 ## How to setup
 
-1. Set up the repository
+> If the CUDA or TensorRT frameworks have already been installed, we strongly recommend uninstalling them first!
 
-If ROS hasn't been installed yet in PC, at first run commands of 1.2 ~ 1.4, described in [ROS wiki](http://wiki.ros.org/melodic/Installation/Ubuntu).
+1. Install ROS (steps taken from the [ROS wiki installation guide](http://wiki.ros.org/melodic/Installation/Ubuntu))

```

Comment:
The previous instructions indicate a smaller subset of installation instructions in the ROS installation guide. I wonder if the rest is covered by the ansible script?

Reply:
This is a good point. I had a look and it seems as if the ansible script installs ROS as well.
https://github.com/tier4/AutowareArchitectureProposal.proj/blob/master/ansible/roles/ros/tasks/main.yml

I think I need to try uninstalling ROS and seeing what happens if I skip these steps and just run the setup script. If it works, then I guess we can remove this first step (and adjust other parts of the installation instructions accordingly).